tests/utils.py

```diff
@@ -176,3 +179,42 @@ async def send_transactions(self, account, calls, nonce=None, max_fee=0):
 
         (call_array, calldata, sig_r, sig_s) = self.signer.sign_transaction(hex(account.contract_address), build_calls, nonce, max_fee)
         return await account.__execute__(call_array, calldata, nonce).invoke(signature=[sig_r, sig_s])
+
+class TestEthSigner():
```

Comment:
does it make sense to move this and the other to a standalone signers module?

Reply:
I vote yes for a standalone signers module